### ISSUE TITLE
Cmake error on Ubuntu: TLU externals found, but could not find!??!?!

### ISSUE THREAD

**Issue by dreyling** [eudaq/eudaq#148]
Trying to configure with TLU producer, cmake complains not finding ZestSC1 package, although it is placed in the extern-folder, as well as, tlufirmware. Even more funny is the log: 1. Found.. 2. could not find. Any ideas??

```
[...]
-- Found ZestSC1 package in 'extern' subfolder: /home/telescope-admin/eudaq/extern/ZestSC1/linux
CMake Warning at cmake/FindZestSC1.cmake:82 (MESSAGE):
  Could not find ZestSC1 driver package required by tlu producer.  Please
  refer to the documentation on how to obtain the software.
Call Stack (most recent call first):
  producers/tlu/CMakeLists.txt:1 (FIND_PACKAGE)


CMake Error at /usr/share/cmake-2.8/Modules/FindPackageHandleStandardArgs.cmake:91 (MESSAGE):
  Could NOT find ZestSC1 (missing: ZESTSC1_LIBRARY)
Call Stack (most recent call first):
  /usr/share/cmake-2.8/Modules/FindPackageHandleStandardArgs.cmake:252 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindZestSC1.cmake:92 (find_package_handle_standard_args)
  producers/tlu/CMakeLists.txt:1 (FIND_PACKAGE)


-- Configuring incomplete, errors occurred!
```

**Comment by TCThenna** [comment]
FindZest cmake script needs to be updated, I'll push it in a second

**Comment by simonspa** [comment]
Pull Request please.

**Comment by TCThenna** [comment]
Seems like I am allowed to push with requesting ...

**Comment by simonspa** [comment]
Please reference tickets in the commit message, this makes it easier to find the solution since they are automatically linked to the ticket.

Something like
```
Bla bla. Fixes #148
```
also automatically closes the issue and marks it as fixed.
